src/Controller/IndexController.php

```diff
@@ -11,10 +13,14 @@ class IndexController extends AbstractController
     /**
      * @Route("/", name="homepage")
      */
-    public function index(): Response
-    {
+    public function index(FigureRepository $figureRepository, ImageRepository $imageRepository): Response
+    {/*
+        $figures = $figureRepository->findAllJoinedToImage();
+        dd($figures);*/
```

Comment:
😄 The comment can possibly stay, but the `dd`...

Reply:
oups... ^^'